CHANGELOG.rst

```diff
@@ -19,7 +19,7 @@
   duplicate the assignment. Thanks `@oliveira-mauricio`_ for the PR.
 - The ``qt_api.extract_from_variant`` and ``qt_api.make_variant`` functions
   (which were never intended for public usage) as well as all class aliases
-  (such as ``qt_bot.QWidget`` or ``qt_bot.QEvent``, among others) are now
+  (such as ``qtbot.QWidget`` or ``qtbot.QEvent``, among others) are now
```

Comment:
gnah, this is still wrong, it should actually be `qt_api`... :see_no_evil: 

Will fix when I edit the changelog.